ci-operator/config/stolostron/olm-addon/.config.prowgen

```diff
@@ -0,0 +1,2 @@
+additional_architectures:
+- arm64
```

Comment:
@droslean should we allow it?

Reply:
We should make sure that our tools will generate the corresponding job by using the correct image instead of https://github.com/openshift/release/pull/41921/files#diff-ba4eaaaa1cee35f47f77662d97d2553f275bfcbb1a80c454001ef0cc7b52b760R82

Reply:
totally fine waiting for a day. For the records this is not something new:
```
release]$ grep -R registry.arm-build01.arm-build.devcluster.openshift.com/ci-arm64/ci-operator:latest | wc -l
512

```